Accessories/uInverter/uInverter Bill of Materials.csv

```diff
@@ -0,0 +1,13 @@
+RefDes,Name,Value,Manufacturer,Manufacturer Part Number,Digi-Key Part Number,Quantity,Description,Comments
```

Comment:
I think this is @npetersen2's comment as well -- but your `value` and `Description` columns do not agree with each other. Please fix.